### PR TITLE
chore(deps): add dependabot bundler

### PR DESCRIPTION
**Pull request by Skarlso**
The bundler PRs are getting out of control. Let's try and rein them in.
This will also add the correct label so the label checker doesn't croak.

kind/feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**:

- [ ] squashed commits into logical changes
- [ ] includes documentation
- [ ] adds unit tests
- [ ] adds or updates e2e tests

### CODE REVIEW

**Comment by Skarlso** [comment]
The resulting PR looks good https://github.com/weaveworks-liquidmetal/flintlock/pull/649

**Review by yitsushi — APPROVED** [review]

**Review by yitsushi — CHANGES_REQUESTED** [review]
just checked a generated PR. That doesn't work
image

**Comment by Skarlso** [comment]
Ah freck.

**Comment by Skarlso** [comment]
I guess I need a way to pass in a signing key to the bot.

**Comment by Skarlso** [comment]
Finalllly! The latest version added verification. I'll release it and use that. Hopefully, it will work. :D

Screenshot 2023-03-04 at 14 14 38

**Comment by Skarlso** [comment]
Screenshot 2023-03-04 at 16 32 19

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -542,6 +543,13 @@ while lower priority or nice-to-have things may take a little longer to get appr
 To help facilitate a smoother and faster review, follow the guidelines [above](#pr-submission-guidelines).
 Submissions which do not meet standards will be de-prioritised for review.
 
+## Dependabot Bundler
+
+There is an action that will periodically bundle dependabot pull requests into a single
```
Comment (Skarlso) [inline CONTRIBUTING.md:548]: I can extend this if more info is required. Maybe with a link to the action once it's available. Or a sample PR once there is one. :)

**Review by yitsushi — APPROVED** [review]

**Pull request merged**